crates/ruff_python_parser/src/lexer/fstring.rs

```diff
@@ -1,27 +1,9 @@
-use bitflags::bitflags;
-
-bitflags! {
-    #[derive(Debug)]
-    pub(crate) struct FStringContextFlags: u8 {
-        /// The current f-string is a triple-quoted f-string i.e., the number of
-        /// opening quotes is 3. If this flag is not set, the number of opening
-        /// quotes is 1.
-        const TRIPLE = 1 << 0;
-
-        /// The current f-string is a double-quoted f-string. If this flag is not
-        /// set, the current f-string is a single-quoted f-string.
-        const DOUBLE = 1 << 1;
-
-        /// The current f-string is a raw f-string i.e., prefixed with `r`/`R`.
-        /// If this flag is not set, the current f-string is a normal f-string.
-        const RAW = 1 << 2;
-    }
-}
+use crate::string_token_flags::StringFlags;
 
 /// The context representing the current f-string that the lexer is in.
 #[derive(Debug)]
 pub(crate) struct FStringContext {
-    flags: FStringContextFlags,
+    pub(crate) flags: StringFlags,
```

Comment:
I think I would prefer to keep this private even at the crate level and provide a method like `flags` or `into_flags` depending on the use-case. The main reason would be to hide the implementation details.